### PR TITLE
Example of a credentials file

### PR DESCRIPTION
**Pull request by ut**
See #253

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -57,15 +57,29 @@ $ bundle exec rails db:seed
...
+```bash
+$ EDITOR=vim bundle exec rails credentials:edit --environment staging
+```
+
+For details on this technique please read this good explainer about [credentials in Rails 6.1](https://blog.saeloun.com/2019/10/10/rails-6-adds-support-for-multi-environment-credentials.html)
+
 ### Optional: Mapbox Token for satellite imagery
 
-You can define your mapbox token in the credentials (token[:mapbox])
+For zoom level 19-21 are no you can addiontanlly use Mapbox satellite imagery as base layer. You'll need a Mapbox account and an API Token. You can define your mapbox token in the credentials (token[:mapbox])
```
Comment (Sefux) [inline README.md:82]: The sentence is not clear to me? 

_For zoom level 19-21 are no you can addiontanlly use Mapbox satellite imagery as base layer._

**Review by Sefux — CHANGES_REQUESTED** [review]
Please check the typo

File: config/credentials.yml.default [hunk 2]
```diff
@@ -0,0 +1,17 @@
+mysql:
```
Comment (Sefux) [inline config/credentials.yml.default:1]: There are no environments set. 
In database.yml they are referenced with environments:
<%= Rails.application.credentials.dig(:mysql, :staging, :password ) %>

**Review by Sefux — CHANGES_REQUESTED** [review]
Check the environments variables

File: README.md (hunk 1 above)
Comment (ut) [reply README.md:82]: Oh yes this to short (and messy) to explain it. How about:


As a default at ORTE, satellite imagery is used as a base layer. This imagery is available only up to level 18. If you want to have satellite imagery on a higher zoom level (19-21), where you can more clearly see details on streets, places and buildings, than you have to define an additional satellite imagery provider. ORTE has as a preset for Mapbox satellite imagery, but to use it, you need to have a Mapbox account and to generate a Mapbox API Token.  (Of course this completely optional, and you switch on user level or permant on map level to a OSM base map.)

....better?

**Pull request merged**